### PR TITLE
[8.0] Various fixes found when deploying v8.0 in prod for LHCb

### PR DESCRIPTION
**Pull request by chaen**
BEGINRELEASENOTES

*Configuration
FIX: getMonitoringBackends returns a list

*WMS
FIX: Matcher handles cases of empty Tag in resource description

*Core
CHANGE: debug log level instead of info in a few places

ENDRELEASENOTES

### CODE REVIEW

**Review by chrisburr — APPROVED** [review]

File: src/DIRAC/WorkloadManagementSystem/Client/Matcher.py [hunk 1]
```diff
@@ -179,7 +179,8 @@ def _processResourceDescription(self, resourceDescription):
             if name in resourceDescription:
                 resourceDict[name] = resourceDescription[name]
 
-        if resourceDescription.get("Tag"):
+        # resourceDescription["Tag"] can be ''
+        if "Tag" in resourceDescription:
```
Comment (andresailer) [inline src/DIRAC/WorkloadManagementSystem/Client/Matcher.py:183]: That is also a problem for 7.3, isn't it?

Comment (chaen) [reply src/DIRAC/WorkloadManagementSystem/Client/Matcher.py:183]: I don't know. I had not seen it before, but I wasn't looking either :D

**Review by fstagni — APPROVED** [review]

File: src/DIRAC/WorkloadManagementSystem/Client/Matcher.py (hunk 1 above)
Comment (fstagni) [reply src/DIRAC/WorkloadManagementSystem/Client/Matcher.py:183]: It could be ported to 7.3, I think. Would you, @andresailer ?

Comment (andresailer) [reply src/DIRAC/WorkloadManagementSystem/Client/Matcher.py:183]: Done: #6713

**Pull request merged**